webhook/utils/transformfilter.go

```diff
@@ -24,7 +27,8 @@ func ValidateFilterTransform(signalType ottl.SignalType, filterSpec []telemetryv
 
 	transformValidator, err := ottl.NewTransformSpecValidator(signalType)
 	if err != nil {
-		return fmt.Errorf("failed to instantiate TransformSpecValidator %w", err)
+		logf.Log.V(1).Error(err, "Failed to instantiate TransformSpec validator")
+		return fmt.Errorf("failed to create pipeline")
```

Comment:
1. Your errors message is a static string - it is better to use `errors.New`.
2. Let's create a private variable since it is used in 2 places.